addons/binding/org.openhab.binding.omnikinverter/src/test/java/org/openhab/binding/omnikinverter/internal/test/OmnikInverterMessageTest.java

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+
+package org.openhab.binding.omnikinverter.internal.test;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.openhab.binding.omnikinverter.OmnikInverterMessage;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class OmnikInverterMessageTest {
```

Comment:
@wborn  does this still apply, I believe in the latest state of the project we use one skeleton which holds main as well as test right?

Reply:
Yes we now add unit tests like these to the same Maven project. When slow/complex integration tests are added, they should be split off to a new project the under `itests`. Preferably most tests should be unit tests for add-ons because these run faster and are easier to maintain. :-)